src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1530,6 +1544,99 @@ def _get_linux_multicontainer_encoded_config_from_file(file_name):
     # Decode base64 encoded byte array into string
     return b64encode(config_file_bytes).decode('utf-8')
 
+def update_deployment_configs(cmd, resource_group_name, name,
+                              deployment_storage_name=None,
+                              deployment_storage_container_name=None, deployment_storage_auth_type=None,
+                              deployment_storage_auth_value=None, slot=None):
+    
+    # NameValuePair = cmd.get_models('NameValuePair')
+    Site, SiteConfig, NameValuePair, AppSettings = cmd.get_models('Site', 'SiteConfig', 'NameValuePair', 'AppSettings')
+    
+    logger.warning("checking the flex app. Params: deployment_storage_container_name: %s", deployment_storage_container_name)
+    is_flex = is_flex_functionapp(cmd.cli_ctx, resource_group_name, name)
+
+    if not is_flex:
+        raise ValidationError("This command is only valid for Azure Functions on the FlexConsumption plan.")
+
+    logger.warning("deployment_storage_name: %s deployment_storage_container_name: %s", deployment_storage_name, deployment_storage_container_name)
+    logger.warning("deployment_storage_auth_type: %s deployment_storage_auth_value: %s", deployment_storage_auth_type, deployment_storage_auth_value)
+
+    if deployment_storage_name == None or deployment_storage_container_name == None:
+        raise ValidationError("--deployment-storage-name and --deployment-storage-container-name are required.")
```

Comment:
These might not be required if the customer just wants to update the storage auth type and value.

Reply:
I have addressed it.